### ISSUE TITLE
[GlobalStep - WooCommerce 4.9 RC 2][IE 11] For Twenty Twenty One theme, "Add to Cart" button labels are not visible on Shop Page

### ISSUE THREAD

**Issue by gglobalstep** [woocommerce/woocommerce#28784]
## Bug Description:
For Twenty Twenty One theme, "Add to Cart" button labels are not visible on Shop Page

## Environment:
Windows 10
Browser: IE11(Version:11.1139.18362.0)
                
 ## Steps To Reproduce:
1. Create any test site using Jurassic Ninja.
1. Install and activate all required plugins.
1. Complete the Onboarding setup wizard.
1. Select Twenty Twenty One theme.
1. Add Some products.
1. Go to frontend and visit to store page.
1. Observe that "Add to Cart" button labels are not visible.

**Note:**
"Add to Cart" button label only appears when mouse is hovered over the button.

## Actual Result:
For Twenty Twenty One theme, "Add to Cart" button labels are not visible on Shop Page
 
## Expected Result:
"Add to Cart" button labels should be visible.

## Screenshots:

![#28784](https://user-images.githubusercontent.com/41110392/104200376-980b6d00-544e-11eb-974e-2fe36a7a18e2.png)

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by juliaamosova** [comment]
Hi @gglobalstep,

Thank you for taking the time to report this bug, we really appreciate your help. I can reproduce it on my end. 

Tested using Browserstack with Windows 10 and EI 11:

![shop_ei](https://user-images.githubusercontent.com/19143190/104219300-ef3c1e00-540b-11eb-83ed-154bdca1d9cb.jpg)

It is still possible to add the product to the cart and when you hover over the button, the text `Add to cart` appears:

![Screen Shot 2021-01-11 at 12 56 05 PM](https://user-images.githubusercontent.com/19143190/104219669-68d40c00-540c-11eb-970c-7fccdbe6ec70.png)

We chatted about this with @vedanshujain and @peterfabian and decided to include it in WC 5.0. 

The issue seems to be related to this change in WP Core: https://core.trac.wordpress.org/ticket/52040.
